user_guide/feature_contexts/defining_step_definitions.rst

```diff
@@ -18,45 +18,10 @@ PHP code called step definitions:
         // do something with $argument
     }
 
-Definitions Home - ``FeatureContext`` Class
--------------------------------------------
-
-Step definitions are just normal PHP methods. They are instance methods in
-a special class called :doc:`FeatureContext</guides/4.contexts>`. This class
-can be easily created by running ``behat`` with the ``--init`` command from
-your project's directory:
-
-.. code-block:: bash
-
-    $ vendor/bin/behat --init
-
-After you run this command, Behat will set up a ``features`` directory
-inside your project:
-
-The newly created ``features/bootstrap/FeatureContext.php`` will have
-an initial context class to get you started:
-
-.. code-block:: php
-
-    // features/bootstrap/FeatureContext.php
-
-    use Behat\Behat\Context\SnippetAcceptingContext;
-    use Behat\Gherkin\Node\PyStringNode;
-    use Behat\Gherkin\Node\TableNode;
-
-    class FeatureContext implements SnippetAcceptingContext
-    {
-        /**
-         * Initializes context.
-         */
-        public function __construct()
-        {
-        }
-    }
+.. note::
 
-All step definitions and :doc:`hooks</guides/3.hooks>` necessary for
-testing your project against your features will be represented as methods inside
-this class.
+   Step definitions are just normal PHP methods. They are instance methods in
+   a special class called :doc:`FeatureContext</user_guide/feature_contexts>`.
```

Comment:
this sentence is wrong. The class can be named the way you want (and you can have multiple context class, in which case you are forced to use different names)